src/plugins/size/size.c

```diff
@@ -0,0 +1,140 @@
+/**
+ * @file
+ *
+ * @brief Source for size plugin
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ *
+ */
+
+#include "size.h"
+
+#include <kdbhelper.h>
+#include <stdio.h>
+#include <string.h>
+
+
+#define DEFAULT_SEPARATOR "x"
+
+static int transform (KeySet * appendKS, const Key * key, const char * sep)
+{
+	char * value = elektraStrDup (keyString (key));
+	if (!value) return 0;
+	char * dptr = strstr (value, sep);
+	if (!dptr) return 0;
```

Comment:
i'm not sure if it should fail. using your example from #1395 
```
[CFontz/Size]
transform/size

[CFontz/Size/Width]
default = 20
range = ...
type = long

[CFontz/Size/Height]
default = 4
range = ...
type = long
```
the value of `Size` would be assumed as invalid/broken/whatever and the default values used instead. imho a warning would be a better

where should we draw the line between a validation and a transformation plugin though ?